sql/mob_groups.sql

```diff
@@ -10632,29 +10632,30 @@ INSERT INTO `mob_groups` VALUES (8761,2633,166,'Metallic_Slime',0,128,1664,0,0,2
 -- Bostaunieux_Oubliette (Zone 167)
 -- ------------------------------------------------------------
 
-INSERT INTO `mob_groups` VALUES (1,6393,167,'Bloodsucker',960,0,302,0,0,65,68,0);
+INSERT INTO `mob_groups` VALUES (1,6393,167,'Bloodsucker',300,0,302,0,0,65,68,0);
 INSERT INTO `mob_groups` VALUES (2,38,167,'Acid_Grease',0,128,15,0,0,52,54,0);
 INSERT INTO `mob_groups` VALUES (3,2764,167,'Mousse',0,128,15,0,0,60,68,0);
-INSERT INTO `mob_groups` VALUES (4,1440,167,'Funnel_Bats',960,0,918,0,0,52,55,0);
-INSERT INTO `mob_groups` VALUES (5,1911,167,'Hecatomb_Hound',960,0,1287,0,0,56,59,0);
-INSERT INTO `mob_groups` VALUES (6,4321,167,'Werebat',960,0,2641,0,0,55,59,0);
-INSERT INTO `mob_groups` VALUES (7,1900,167,'Haunt',960,0,1281,0,0,60,63,0);
-INSERT INTO `mob_groups` VALUES (8,6526,167,'Garm',960,0,933,0,0,64,66,0);
-INSERT INTO `mob_groups` VALUES (9,908,167,'Dark_Aspic',960,0,567,0,0,52,54,0);
+INSERT INTO `mob_groups` VALUES (4,1440,167,'Funnel_Bats',300,0,918,0,0,52,55,0);
+INSERT INTO `mob_groups` VALUES (5,1911,167,'Hecatomb_Hound',300,0,1287,0,0,56,59,0);
+INSERT INTO `mob_groups` VALUES (6,4321,167,'Werebat',300,0,2641,0,0,55,59,0);
+INSERT INTO `mob_groups` VALUES (7,1900,167,'Haunt',300,0,1281,0,0,60,63,0);
+INSERT INTO `mob_groups` VALUES (8,1471,167,'Garm',300,0,933,0,0,64,66,0);
```

Comment:
Generally, conflicts should be resolved with rebases, not merges. If a Pull Request has a merge commit in its history, the PR should be held until it has a clean commit history, because it is likely community members who try to resolve conflicts with merges will botch the process.

This is what happened here - @wrenffxi [specifically created a new pool for the Garms in Bostaunieux - and other groups in Bostaunieux - as part of their Undead QC PR](https://github.com/project-topaz/topaz/commit/c1b010a4dc57c4bb9a03b7a1561832ee7e850dc3#diff-2c6729e449bb65fc41e44ba3d47356ec). This, along with other work on the mob groups in the zone, has been undone.

[The outdated version of the branch did not have the new group values](https://github.com/project-topaz/topaz/pull/952/commits/cbcba185ccd95a97fabf36ae4de3bd1356d6cfcc). The conflicting line in the "updated" file was taken in its entirety. This is _not_ how merges should be done. **Merge conflicts happen for a reason**. Two people touched the same line - it is vital to determine _what_ caused the conflict so that the merged line incorporates _both_ changes.

Reply:
You didnt look at the entire PR, as I did resolve the conflict

Reply:
OK i had a discussion in discord, and I think i understand why rebase is prefered.
The way we work with git at work is that we merge all changes from a PR into a single commit, so there would be no difference

Reply:
You selected your line change in its entirety to be the final line in the resulting file. This makes git stop complaining about the conflict - it does not actually resolve it.

Reply:
Ok i understand the confusion. I thought we were discussing mob_droplist because thats where the merge conflict was.
Not sure why but I had no merge conflict on mob_groups. But I see that I need to fix what I have undone...